### ISSUE TITLE
Support for all post types

### ISSUE THREAD

**Issue by grappler** [xwp/wp-customize-object-selector#35]
It was raised in another PR that `'post_type' => 'any'` should be supported.
https://github.com/wp-shortcake/shortcake/pull/736/files#r117840102

When I tried it out in this plugin I got "Bad post type" because `any` is not a valid post type but a valid query.

This comes from `get_post_type_object( $post_type )` in https://github.com/xwp/wp-customize-object-selector/blob/master/php/class-plugin.php#L421

This would also affect `$is_multiple_post_types = count( $query->get( 'post_type' ) ) > 1;`

**Comment by westonruter** [comment]
Yeah, you're right. I suppose if `$post_query_vars['post_type']` contains `any` we should iterate over all registered post types to get each one that the user has privileges to `read` and then add them to the array of post types, and remove “any”.
